layouts/partials/hooks/body-end.html

```diff
@@ -7,28 +7,63 @@
 {{/* copy-and-paste helper for codenew shortcode */}}
 {{- if or (.HasShortcode "code_sample") (.HasShortcode "code") (.HasShortcode "codenew") -}}
   {{- if hugo.IsProduction -}}
-    {{- $sweetAlert :=  resources.Get "js/sweetalert-2.1.2.min.js" | minify | fingerprint -}}
-<script async src="{{ $sweetAlert.RelPermalink }}" integrity="{{ $sweetAlert.Data.Integrity }}" crossorigin="anonymous"></script>
+    {{- $toastrCss := resources.Get "css/toastr-2.1.4.min.css" -}}
+    <LINK rel="stylesheet" href="/css/toastr-2.1.4.min.css">
+    {{- $toastrJs := resources.Get "js/toastr-2.1.4.min.js" | minify | fingerprint -}}
+    <script async src="{{ $toastrJs.RelPermalink }}"></script>
   {{- else -}}
-    {{- $sweetAlert := resources.Get "js/sweetalert-2.1.2.min.js" -}}
-<script async src="{{ $sweetAlert.RelPermalink }}"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css" rel="stylesheet">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
   {{- end -}}
+
+  <style>
+    .toast-success {
+      background-color: #007bff !important; 
+      color: #ffffff !important; 
+    }
+  </style>
```

Comment:
- Use Kubernetes blue (`$primary` in Sass)
- Put this code in `assets/scss/_custom.scss` instead of here.